templates/_base.html

```diff
@@ -185,7 +181,8 @@
     // This happens when an anon visitor tries to view a page that
     // requires being signed in.  We cannot promptSignIn(), because
     // lacking a user-initiated event, the popup will be blocked.
-    if (getQueryStringValue("loginStatus") == 'False') {
+    const urlParams = new URLSearchParams(window.location.search);
+    if (urlParams.get("loginStatus") == 'False') {
```

Comment:
```suggestion
    if (urlParams.get("loginStatus") == null) {
```
The boolean in javascript should be true and false but here specifically we should use `null`, per. https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/get

Reply:
It's actually set as a string spelled out as "False" in the URL rather than being a boolean.  

It's defined here
https://github.com/GoogleChrome/chromium-dashboard/blob/main/settings.py#L67
and used here and a few other places
https://github.com/GoogleChrome/chromium-dashboard/blob/main/framework/permissions.py#L97

Reply:
aha I see